test/JDBC/input/BABEL-2795-vu-prepare.sql

```diff
@@ -0,0 +1,64 @@
+CREATE VIEW BABEL_2795_vu_prepare_v1 AS (select datediff(dd,DATETIMEFROMPARTS(2021,12,10,1,50,59,0),DATEADD(minute,123456*1440,DATETIMEFROMPARTS(1900,1,1,0,0,0,0))));
```

Comment:
can we add more variations for 1 Jan, 28 Feb, 29 Feb, leap years and first data is bigger than second etc.

Reply:
Thanks for checking for the test cases Surendra. This is why we should have the test checklist template and try to cover all the checklist.

Reply:
Added all such testcases related to leap year